### ISSUE TITLE
Source request: HoneyToon

### ISSUE THREAD

**Issue by Jefferson24323** [keiyoushi/tachiyomi-extensions-source#6362]
### Source name

Honey toon

### Source link

https://honeytoon.com/pt

### Source language

Português 

### Other details

18+NSFW= sim

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by Norsze** [comment]
This is not a Português source, it is an English one with some titles in Português too

**Comment by mouakka** [comment]
Mind me asking it it does exist if not when will it be. Regards.

**Comment by DinilHansana** [comment]
Any Updates on this?

**Comment by Smol-Ame** [comment]
No, there's no updates on this. If a contributor wants to spend their own time & efforts into making this into an extension, they can do it & submit a pull request. There's over 670~ requests in the repo, with plenty that's older than this one, and only a handful of contributors.

Don't comment if you don't have any useful information that could help contributors out. It unnecessarily pings the Keiyoushi Discord channel and emails everyone subscribed to the request. Wait patiently, contribute yourself, or find somewhere that will work on demand for free for you.
